apps/macos/src/Podfile

```diff
@@ -24,8 +24,8 @@ use_test_app! do |target|
   target.app do
     platform :osx, '10.14'
 
-    pod 'FluentUI-React-Native-Button', :path => '../../../packages/experimental/NativeButton/FluentUIReactNativeButton.podspec'
     pod 'FluentUI-React-Native-Avatar', :path => '../../../packages/experimental/Avatar/FluentUIReactNativeAvatar.podspec'
+    pod 'FluentUI-React-Native-Button', :path => '../../../packages/experimental/NativeButton/FluentUIReactNativeButton.podspec'
     pod 'FluentUI-React-Native-FocusZone', :path => '../../../packages/components/FocusZone/FluentUIReactNativeFocusZone.podspec'
     pod 'FluentUI-React-Native-Radio-Button', :path => '../../../packages/components/NativeRadioButton/FluentUIReactNativeRadioButton.podspec'
     pod 'FluentUI-React-Native-MenuButton', :path => '../../../packages/components/MenuButton/FluentUIReactNativeMenuButton.podspec'
```

Comment:
Looks like you alphabetized the Button which is awesome. Might as well fix up the rest of them while we're here (e.g. Menu is before Radio)